### PR TITLE
added encoding to correct encoding problem related to issue #3461

### PR DESCRIPTION
**Pull request by INOPIAE**

### CODE REVIEW

File: Rubberduck.VBEEditor/SafeComWrappers/VBA/VBComponent.cs [hunk 1]
```diff
@@ -114,7 +114,7 @@ private void ExportUserFormModule(string path)
 
             var tempFile = ExportToTempFile();
             var tempFilePath = Directory.GetParent(tempFile).FullName;
-            var contents = File.ReadAllLines(tempFile);
+            var contents = File.ReadAllLines(tempFile,System.Text.Encoding.UTF7);
```
Comment (Vogel612) [inline Rubberduck.VBEEditor/SafeComWrappers/VBA/VBComponent.cs]: are you sure that UTF-7 is the correct encoding? I'd have expected some system-dependent codepage... (also space after comma) :)

Comment (INOPIAE) [reply Rubberduck.VBEEditor/SafeComWrappers/VBA/VBComponent.cs]: I tested different encodings. With utf-8 the error documented in #3641 did not show up any more.

**Comment by retailcoder** [comment]
Closes #3461

**Comment by Vogel612** [comment]
We're currently assuming that this PR causes #3518   
What are our options?

**Pull request merged**